### ISSUE TITLE
i3 freezes when I drag a chrome tab to a new window that is assigned to a different workspace

### ISSUE THREAD

**Issue by japhir** [i3/i3#2274]
Output of `i3 --moreversion 2>&- || i3 --version`:
Binary i3 version:  4.12 (2016-03-06, branch "4.12") © 2009 Michael Stapelberg and contributors
Running i3 version: 4.12 (2016-03-06, branch "4.12") (pid 1605)
Loaded i3 config: /home/japhir/.config/i3/config (Last modified: Sun 27 Mar 2016 10:33:11 PM CEST, 55538 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3

URL to a logfile as per http://i3wm.org/docs/debugging.html:
http://logs.i3wm.org/logs/5632581307465728.bz2

**What I did:**
I'm dragging a tab named "WhatsApp Web" from workspace 2. In my config a window of class chromium with name "WhatsApp Web" is assigned to workspace 4.

**What I saw:**
The tab is moved up to the point where it should become a window but then i3 freezes until I restart it.

**What I expected instead:**
Releasing the tab -> window should result in it being moved to workspace 4.

**Comment by Airblader** [comment]
Does i3 freeze or Chrome? As in, can you still switch to other workspaces?

**Comment by japhir** [comment]
I've just experimented a little.

It seems like the workspace that has the old chrome on it freezes (doesn't respond to any mouse input) but switching workspaces with the keyboard still works. 
If I switch back to the old workspace it doesn't render properly, showing the other workspace I just visited. 
Furthermore, if the workspace I'm automagically sending the new window to is already in focus it does not freeze. If it's open in the background it shows the red notification thingie but the other workspace is still frozen until the new workspace is in focus (obviously only reproducible with multiple monitors).
I hope this helps identify where the problem lies :)
